core/Makefile

```diff
@@ -203,11 +203,13 @@ Enclave/enclave_hsm_t.c: $(SGX_EDGER8R) Enclave/enclave_hsm.edl
 	@cd Enclave && $(SGX_EDGER8R) --trusted ../Enclave/enclave_hsm.edl --search-path ../Enclave --search-path ../Enclave/self_test --search-path $(SGX_SDK)/include --search-path $(OPENSSL_PATH)/include
 	@echo "GEN  =>  $@"
 
+Enclave/enclave_hsm_t.h: Enclave/enclave_hsm_t.c
```

Comment:
Thanks. Existing codes are not standard. Could we change Line 202 to:
Enclave/enclave_hsm_t.**h** : .....
And update your line to:
Enclave/enclave_hsm_t.c: Enclave/enclave_hsm_t.h

Reply:
Ok, of course

Reply:
I've updated commit